app/src/main/res/values/strings.xml

```diff
@@ -486,5 +486,6 @@
     <string name="category_string">Category</string>
     <string name="connection">Connected</string>
     <string name="contributor_string">Contributor</string>
+    <string name="app_disabled_text">Application Is Disabled</string>
```

Comment:
Please remove capitalisation, "Application" should only start with a capital

Reply:
"The application is disabled."